src/polycubed/src/CMakeLists.txt

```diff
@@ -65,7 +65,7 @@ set(polycubed_sources
   cubes_dump.cpp
   ${server_sources}
   ${CMAKE_CURRENT_BINARY_DIR}/load_services.cpp
-  ${CMAKE_CURRENT_BINARY_DIR}/version.cpp)
+  ${CMAKE_CURRENT_BINARY_DIR}/version.cpp server/Resources/Endpoint/Hateoas.cpp server/Resources/Endpoint/Hateoas.h)
```

Comment:
I would suggest to list these file one per line, e.g., after line 65.

Reply:
Fine, moved to Endpoint/CMakeLists.txt